packages/intrepid2/unit-test/Discretization/Integration/test_05.hpp

```diff
@@ -139,16 +141,16 @@ namespace Intrepid2 {
 
       // compute and compare integrals
       try {
-        const auto maxDeg   = Parameters::MaxCubatureDegreeTri;
+        const auto maxDeg   = 10; //Parameters::MaxCubatureDegreeTri;
```

Comment:
Why did you change this?

Reply:
It takes too much time. Now we do not use UVM, all test verification includes explicit deep copy and a lot more kernels launches. This is all work in progress and we will consider to test all when we complete the uvm removal.

Reply:
In the next PR, I will only lower the order for cuda and host code will do test the max.